LambdaEngine/Source/Game/ECS/Systems/Rendering/AnimationSystem.cpp

```diff
@@ -126,15 +130,27 @@ namespace LambdaEngine
 		// Animation system has its own clock to keep track of time
 		m_Clock.Tick();
 
+		TArray<uint32> jobIndices;
```

Comment:
You could preallocate memory for the array. I know you don't start a thread for paused animations, so the preallocation will be excessive. But memory storage is cheap, and heap allocations aren't (relatively speaking).

Reply:
Will fix